docs/annotations.md

```diff
@@ -213,6 +213,55 @@ spec:
           servicePort: 80
 ```
 
+## AppGW Trusted Root Certificate
+Users can now [configure their own root certificate to Application Gateway](https://docs.microsoft.com/en-us/cli/azure/network/application-gateway/root-cert?view=azure-cli-latest) to be trusted via AGIC.
+The annotaton `appgw-trusted-root-certificate` shall be used together with annotation `backend-protocol` to indicate end-to-end ssl encryption, mulitple root certificates, seperated by comma, if specified, e.g. "name-of-my-root-cert1,name-of-my-root-certificate2"
+
+### Use Azure CLI to install your root certificate to Application Gateway
+* Create your public root certificate for testing
+```bash
+openssl ecparam -out test.key -name prime256v1 -genkey
+openssl req -new -sha256 -key test.key -out test.csr
+openssl x509 -req -sha256 -days 365 -in test.csr -signkey test.key -out test.crt
+```
+
+* Rename test.crt to test.cer
+
+* Configure your root certificate to Application Gateway
+```bash
+az network application-gateway root-cert create --cert-file test.cer  --gateway-name $appgwName  --name name-of-my-root-cert1 --resource-group $resgp
+```
+* Repeat the steps above if to configure multiple trusted root certificates
```

Comment:
Repeat the steps above if you want to configure multiple trusted root certificates.